src/utils/AccessibleButton.ts

```diff
@@ -154,19 +153,22 @@ export class AccessibleButton {
     }
   }
 
-  private ensureCorrectLabel() {
-    if (!this.hasLabel) {
+  private ensureCorrectLabelOrTitle() {
+    if (!this.hasLabelOrTitle) {
```

Comment:
I think the name for the boolean variable here should be `shouldPrioritizeTitleOverLabel`?
With the current name, it's not clear to me what the difference is between this `if` condition and the `if` on line 161.

Reply:
Name will be changed to `labelOrTitleRequired`
Here the condition verifies if the dev removed the rule.

At line 161, we check that the rule is respected.